exercises/src/main/scala/fpinscala/parallelism/Par.scala

```diff
@@ -3,39 +3,213 @@ package fpinscala.parallelism
 import java.util.concurrent._
 
 object Par {
+  /*
+  Par[A] needs to be a type that can contain a computation of type A and
+  also yield the value when asked for it. It also needs to contain a
+  marker for whether its computation is concurrent or sequential. So one
+  possible implementation is:
+
+  case class Par[A](comp: A, concurrent: Boolean)
```

Comment:
You couldn't have a definition of `Par[A]` that has a field of type `A`, because it represents a computation of a value that is initially unknown. So instead the type `Par[A]` should be or contain a function that return a value of type A, or another container of type `A`.